package-lock.json

```diff
@@ -91975,7 +91975,7 @@
     },
     "packages/libs": {
       "name": "@audius/sdk",
-      "version": "3.0.10-beta.4",
+      "version": "3.0.10-beta.9",
```

Comment:
feels like this should be added to package-lock.json. cc @dylanjeffers

Reply:
Yeah agreed, not sure what happened there

Reply:
fast! and cool thx